riscv-asm.md

```diff
@@ -545,6 +545,23 @@ referenced on [this page](https://en.cppreference.com/w/cpp/language/floating_li
 Load Floating-point Immediate
 -------------------
 
+The `Zfa` extension introduces `fli.{h|s|d|q}` instruction for loading a specific
+set of floating-point immediates, supported value can be found from the RISC-V
```

Comment:
value => values. Though we list all the values below, so perhaps "supported values can be found in the RISC-V ISA specification but are also listed below."